tests/default/_core/search/pipeline.yaml

```diff
@@ -0,0 +1,33 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+description: Test the creation of a search pipeline.
+version: '>= 2.8'
```

Comment:
probably here it should be > 2.8 ?
https://github.com/opensearch-project/opensearch-api-specification/blob/ebe0f8a885f7db7e882d160c101055a5aa70a707/spec/namespaces/search_pipeline.yaml#L11